learnEcl/2400-set.md

```diff
@@ -2,3 +2,99 @@
 title: Set
 slug: set
 ---
+
+# SET
+
+SET converts values from a field/column in a dataset to a SET OF. SET can also be defined in brackets [ ] without need of a dataset. Since data isn't distributed it has to fit in memory.
```

Comment:
The sentence about defining a SET with [] is good, but needs to be in the SET OF section because you're really creating a static SET.  Basically, there is no relationship between SET() and a static SET OF created with [].